internal/testutils/tar.go

```diff
@@ -7,12 +7,17 @@ import (
 	"testing"
 )
 
+const expectedUidAndGid = 0
+
 type ExpectedFile struct {
 	Path     string
 	Contents string
 
 	// If true, we will assert that the file is not in the tarball.
 	Missing bool
+
+	// If true, we will assert that UID and GIF are 0
+	AssertUidAndGidAreZero bool
```

Comment:
should we just assert this all the time?

Reply:
No because we use this in other tests where we assert that files exist in docker images.

I found that out the hard way because that was my first approach, heh.